recipes/pytest-pyvista/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "pytest-pyvista" %}
+{% set version = "0.1.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/p/pytest_pyvista/pytest_pyvista-{{ version }}.tar.gz
+  sha256: 43e6f0a85b6ff23aff51b7f750528de0fdda6da2011a94b636f05d8498fe38a0
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+  run:
+    - python >=3.8
+    - pytest >=3.5.0
+    - pyvista
+    - flit_core >=3.2,<4
```

I think this needs to be flit
```suggestion
    - flit >=3.2,<4
```

---

It also might have to go in host?  `setuptools` is mentioned as being put in host in the documentation.  https://conda-forge.org/docs/maintainer/adding_pkgs.html#build

---

@tkoyama010 This is no need to have `flit` or `flit_core` in the `run` section 👍

---

```suggestion
```